### DISCUSSION TITLE
Update to Japanese locale [ja]

### DISCUSSION THREAD

**Discussion by yuji-tmr** [aclap-dev/video-downloadhelper#642] [Language & Translation]
i have translated for Japanese locale

[ja.txt](https://github.com/aclap-dev/video-downloadhelper/files/15443653/ja.txt)

**Comment by paulrouget** [comment]
Thank you so much! We will add your translation in the upcoming version :)

**Comment by paulrouget** [comment]
This will be in v9.1.
